tests_python/test_debugger.py

```diff
@@ -3109,16 +3109,6 @@ def test_remote_debugger_threads(case_setup_remote):
         writer.write_run_thread(hit_in_main.thread_id)
         writer.write_run_thread(hit_in_thread1.thread_id)
         writer.write_run_thread(hit_in_thread2.thread_id)
-
-        if TODO_PY312:
```

Comment:
This code was only necessary because of the extra breakpoints firing with the line/jump events. With my fix in place this is no longer needed.